recipe/meta.yaml

```diff
@@ -21,29 +23,47 @@ requirements:
   run:
     - python
     - six
-    - scandir  # [py27]
+    - scandir  # [py<35]
+    - typing  # [py<35]
```

Comment:
```suggestion
  run_constrained:
    - scandir  # [py<35]
    - typing  # [py<35]
```
Move them to `run_constrained` because they are optional dependencies https://github.com/jazzband/pathlib2/blob/2.3.7-post1/setup.py#L45-L47